docs/content/direct/authorization.md

```diff
@@ -0,0 +1,81 @@
+- Do this per WEC, using your preferred process/tooling.
+
+Example (grant access to custom resources):
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: example-crd-access  # Use a descriptive name for your use case
+rules:
+- apiGroups: ["example.my.group"]  # Replace with your actual API group
+  resources: ["widgets"]           # Replace with your custom resources
+  verbs: ["get", "list", "watch", "create", "update", "patch"]  # Recommendation: add only verbs you need
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: klusterlet-example-crd-access  # Should identify your ClusterRole purpose
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: example-crd-access  # Must match the ClusterRole name above
+subjects:
+- kind: ServiceAccount
+  name: klusterlet-work-sa           # This is the OCM work agent service account
+  namespace: open-cluster-management-agent  # This is the OCM agent namespace
+```
+
+2) Downsync RBAC objects
+- Treat RBAC as part of your desired state in the WDS and downsync the `ClusterRole`/`(Cluster)RoleBinding` objects to selected WECs via a `BindingPolicy`.
+- This keeps RBAC changes auditable and repeatable in the same pipeline you use for the rest of your workload.
+- Note that this approach is a specific example of approach 1, where KubeStellar serves as your preferred process/tooling for managing RBAC.
+
+Tip: OCM also supports an aggregated ClusterRole pattern. If you create a `ClusterRole` labeled `open-cluster-management.io/aggregate-to-work: "true"`, its rules are automatically included in the existing `klusterlet-work-sa` permissions through OCM's aggregation mechanism. You can also downsync such a `ClusterRole`. With the aggregated pattern, no separate ClusterRoleBinding is needed - OCM handles the permission binding automatically.
+
+Example (aggregated ClusterRole):
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: example-open-cluster-management:klusterlet-work:extra-rules  # Use descriptive naming
+  labels:
+    open-cluster-management.io/aggregate-to-work: "true"  # This label enables auto-aggregation
+rules:
+- apiGroups: ["example.my.group"]  # Replace with your actual API group
+  resources: ["widgets"]           # Replace with your custom resources
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]  # Include needed verbs
+```
+
+Refer to OCM's docs for details on both methods: [Permission setting for work agent](https://open-cluster-management.io/docs/concepts/work-distribution/manifestwork/#permission-setting-for-work-agent)
+
+## Concrete example: Out-of-tree CRDs
+
+Scenario 2 shows how to grant the agent the additional rights needed to manipulate an out-of-tree CRD (AppWrapper). See [Scenario 2](./example-scenarios.md#scenario-2---out-of-tree-workload) in the Example Scenarios.
```

Comment:
I just noticed that GitHub and mkDocs are using different fragment identifers here!